### DISCUSSION TITLE
Is there a way to keep highlights even after changing the name of the file?

### DISCUSSION THREAD

**Discussion by nickel8448** [ahrm/sioyek#1113] [Q&A] [answered]
Hello,

I would like to keep highlights in a file even after I change the name of the file. I understand that a hash of the file is calculated and the hash value is used as a key to store the highlights. I do not want to modify the database manually, is there any other way?

Thanks!

**Comment by ahrm** [comment, marked as answer]
Just renaming (or moving) the file should not change the hash. Do the highlights not appear if you restart sioyek?

**Comment by nickel8448** [comment]
They do not appear :(

My file name was "(Pending) Aristotle.pdf" and I changed it to "(Read) Aristotle.pdf" and all my highlights were gone.

**Comment by nickel8448** [comment]
Apologies, restarting Sioyek did help. I hardly quit the reader so did not realise to do that yet.
